relay-server/src/utils/rate_limits.rs

```diff
@@ -85,7 +85,7 @@ pub fn parse_rate_limits(scoping: &Scoping, string: &str) -> RateLimits {
 ///   to be set on the event item.
 /// - `Attachment`: If the attachment creates an event (e.g. for minidumps), the category is assumed
 ///   to be `Error`.
-fn infer_event_category(item: &Item) -> Option<DataCategory> {
+pub fn infer_event_category(item: &Item) -> Option<DataCategory> {
```

Comment:
Rather than simply adding `pub`, would it be better to move this elsewhere? (With this change, it's arguably not strictly scoped to the concern of rate limits.)

Reply:
For simplicity, you can totally keep this here for this changeset. We'll likely have to touch this in a follow-up and create more utilities like this, at which point code organization will become more clear.